packages/math-input/src/components/tabbar/__tests__/tabbar.test.tsx

```diff
@@ -1,65 +1,54 @@
-import {mount} from "enzyme";
+import {render, screen} from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
 import * as React from "react";
 
 import Tabbar from "../tabbar";
 
 describe("<Tabbar />", () => {
-    it("defaults to selecting the first item", () => {
+    it("renders the numbers page", () => {
         // Arrange
-        const wrapper = mount(
+        render(
             <Tabbar
-                items={["Numbers", "Geometry", "Operators"]}
-                onSelect={() => {}}
+                items={["Numbers"]}
+                selectedItem={"Numbers"}
+                onSelectItem={() => {}}
             />,
         );
 
         // Assert
-        expect(wrapper).toHaveState("selectedItem", 0);
-        const firstItem = wrapper.find("TabbarItem").first();
-        expect(firstItem).toHaveProp("itemState", "active");
+        expect(screen.getByRole("button", {name: "Numbers"}));
```

Comment:
This doesn't seem to be asserting what the test name says.  We're asserting that the tab bar rendered the `Numbers` tab item, but no mention of the contents of the Numbers page.  Shouldn't we assert the some known component on the first tab page is rendered?

Reply:
Fixed 👍